contracts/Relayer.sol

```diff
@@ -44,6 +46,11 @@ contract Relayer is IRelayer {
         _;
     }
 
+    modifier _onlyBridge() {
```

Comment:
If we're to use this modifier, then the Bridge address needs to be set after deployment, because the Bridge contract requires the Relayer address as a constructor argument. As this contract stands now, any method using the modifier is uncallable because `_bridgeAddress` never gets set

Reply:
I think the Relayer contract should check if the sender is the owner instead of the Bridge